tasks/libs.yml

```
@@ -78,40 +63,28 @@
 
 - name: 'Link to jcl-over-slf4j library'
   file:
-    name: '{{ jetty.base }}/lib/logback/jcl-over-slf4j.jar'
+    name: '{{ jetty.base }}/lib/logging/jcl-over-slf4j-{{ download.jcl_over_slf4j.version }}.jar'
     src: '{{ shib_idp.libs }}/jcl-over-slf4j-{{ download.jcl_over_slf4j.version }}.jar'
     state: link
 
-- name: 'Install slf4j-api'
+- name: 'Install aaf-shibext library'
   get_url:
-    url: '{{ urls.slf4j_api.url }}'
-    dest: '{{ shib_idp.libs }}/slf4j-api-{{ download.slf4j_api.version }}.jar'
-    checksum: 'sha256:{{ download.slf4j_api.sha256sum }}'
+    url: '{{ urls.aaf_shib_ext.url }}'
+    dest: '{{ shib_idp.home }}/edit-webapp/WEB-INF/lib/aaf-shibext-{{ download.aaf_shib_ext.version }}.jar'
```

I guess, not a typo, given it appears twice
```suggestion
    dest: '{{ shib_idp.home }}/edit-webapp/WEB-INF/lib/aaf-shib-ext-{{ download.aaf_shib_ext.version }}.jar'
```

---

Not a typo.